kubernetes-ingress-defaultbackend.yaml

```diff
@@ -22,7 +22,7 @@ pipeline:
     with:
       repository: https://github.com/kubernetes/ingress-gce
       tag: v${{package.version}}
-      expected-commit: 99d9b307c1dc475b6bdd345a4055f1ba3228a102
+      expected-commit: 057cd165dbbb9a770a69b39190a300acd54cfede
```

Comment:
Looks like this package has been upgraded to version 1.31.6 recently. You'll want to update this file (`version`, `epoch`, and `expected-commit` to match:
 https://github.com/wolfi-dev/os/blob/main/kubernetes-ingress-defaultbackend.yaml#L1-L25